### PR TITLE
Add SD card content to SDBlockDevice.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Add content about wiring an SD card module to a developer board.

### CODE REVIEW

File: docs/api/storage/SDBlockDevice.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ You can use the Mbed OS SD card block device, so applications can read and write
 
 To configure this class, please see our [BlockDevice configuration documentation](../reference/storage.html#blockdevice-default-configuration).
 
+## How to wire an external SD-card module to a developer board
+
+<span class="images">![](../images/sd-wiring.png)<span>How to wire an external SD-card module to a developer board</span></span>
```
Comment (teetak01) [inline docs/api/storage/SDBlockDevice.md]: Should this PR also include the .png or how does that work?

Comment (AnotherButler) [reply docs/api/storage/SDBlockDevice.md]: I added the PR separately: https://github.com/ARMmbed/mbed-os-5-docs/tree/development/docs/images

**Review by teetak01 — COMMENTED** [review]
Should this also mention that application needs to override the SD pin mapping based on the wiring?

For. ex. as is done in https://github.com/ARMmbed/mbed-cloud-client-example/blob/master/mbed_app.json#L123-L126

File: docs/api/storage/SDBlockDevice.md (hunk 1 above)
Comment (teetak01) [reply docs/api/storage/SDBlockDevice.md]: Ok, a bit unclear why you would do those in seperate PRs.

**Comment by VeijoPesonen** [comment]
Not to forget that the "SD" component needs to be added - [Component SD](https://github.com/ARMmbed/mbed-cloud-client-example/blob/fd495ae5d9c94e3501cc17bc2a71d51ed1487179/mbed_app.json#L109)

**Comment by AnotherButler** [comment]
@VeijoPesonen and @teetak01 Do these changes look correct?

**Review by VeijoPesonen — APPROVED** [review]

**Review by teetak01 — APPROVED** [review]

**Pull request merged**